lib/compute/jenkins-main-node.ts

```diff
@@ -158,6 +166,7 @@ export class JenkinsMainNode {
       InitPackage.yum('httpd'),
 
       // Configuration to proxy jenkins on :8080 -> :80
+      // Configuration changes depending on useSsl flag
```

Comment:
If you feel a comment is valuable consider something like `jenkins proxy config redirects from port 80 -> 443 when ssl is enable to force traffic to be secure".  This explains the why rather than the what.  Comments that inform exactly what is written in the code are a form of duplication.

See this great blog post from SO for more about writing comments https://stackoverflow.blog/2021/07/05/best-practices-for-writing-code-comments/